index.d.ts

```diff
@@ -12,7 +12,8 @@ export const tools: { env: { inject: (filename?: string) => void } };
 
 // see: https://github.com/zapier/zapier-platform-cli/issues/339#issue-336888249
 export const createAppTester: (
-  appRaw: object
+  appRaw: object,
+  storeKey: string
```

Comment:
very good guess! The only change is that it should be `storeKey?: string` because it's optional.

Reply:
👍